v0/destinations/fbpixel/transform.js

```diff
@@ -0,0 +1,431 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable camelcase */
+const sha256 = require("sha256");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+const { EventType } = require("../../../constants");
+
+const {
+  constructPayload,
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  flattenJson
+} = require("../../util");
+
+function formatRevenue(revenue) {
+  return Number((revenue || 0).toFixed(2));
+}
+
+function getContentType(message, defaultValue, categoryToContent) {
+  const { options } = message;
+  if (options && options.contentType) {
+    return [options.contentType];
+  }
+
+  let { category } = message.properties;
+  if (!category) {
+    const { products } = message.properties;
+    if (products && products.length) {
+      category = products[0].category;
+    }
+  }
+  if (category) {
+    if (categoryToContent === undefined) {
+      categoryToContent = [];
+    }
+    const mapped = categoryToContent;
```

Comment:
write a comment of what the block does

Reply:
something like "mapping properties.category to user provide content else taking the default value as per ecomm spec"

Reply:
done